README.md

```diff
@@ -15,6 +15,8 @@ cd bahmni-docker
 export OPENMRS_MODULES_PATH="/path/to/the/openmrs/modules"
 export BAHMNI_CONFIG_PATH="/path/to/the/bahmni/config"
 export OPENMRS_CONFIG_PATH="/path/to/the/openmrs/config"
+export BAHMNI_HOME="/path/to/the/bahmni/home"
+export TIMEZONE="some/timezone" #Example for IST set TIMEZONE="Asia/Kolkata"
```

Comment:
Unless I am mistaken, I believe this env var is used at build time right?
That should probably be mentioned. In particular, one should enforce the `--build` argument in order to update the Time Zone of the container.